ROADMAP.adoc

```diff
@@ -0,0 +1,77 @@
+[[minishift-roadmap]]
+= Minishift Roadmap
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[vision]]
+== Vision
+
+*Run OpenShift locally.*
+
+Minishift's goal is to become the default choice for running a
+single-node OpenShift cluster on your local host. Either for evaluation
+purposes or for ongoing development.
+
+Due to the fact that Minishift builds on OpenShift
+https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md[cluster up],
+Minishift's focus lies on providing more value to the basic
+_cluster up_ provisioning. This includes virtual machine creation and
+management, making it easier to configure OpenShift in precisely the way
+you want and providing useful shortcut commands for making your work
+easier.
+
+[[issue-tracking]]
+== Issue tracking
+
+The Minishift team currently uses the GitHub
+https://github.com/minishift/minishift/issues[issue tracker] to plan its
+work. Issues are generally categorized into __tasks__, _bugs_ and
+__features__. Looking at the latter,
+__https://github.com/minishift/minishift/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Ffeature[features]__,
+is probably the best way to get an overview where Minishift is heading.
+
+[[upcoming-versions]]
+== Upcoming versions
+
+The following sections give a high level overview of the envisioned
+features and how they fit into upcoming releases. Of course this is
+subject to change and actual version names might change as well, because
+http://semver.org/[semantic versioning] is used to determine the final
+version name of a given release.
+
+[[minishift-100]]
+=== Minishift 1.0.0
+
+- Ability to select the OpenShift version of the cluster [line-through]#link:https://github.com/minishift/minishift/issues/316[#141]#
```

Comment:
You lost some formatting here. In Markdown completed features used strikethrough. Now there is no indication anymore what is completed. If not strikethrough, we might now use checkbox '- [ ] ' syntax which is quite nice as well. All expect "Consistent host folder mounting https://github.com/minishift/minishift/issues/316[#316], https://github.com/minishift/minishift/issues/317[#317]" should be marked completed.